setup.py

```diff
@@ -329,7 +330,7 @@ def run(self):
     "backend": [
         "jinja2==2.11.2",
         # PostgreSQL
-        "triopg==0.3.0",
+        "triopg==0.5.0",
```

Comment:
What is the reason to bump `triopg` version ?

Reply:
Because of the trio bump, which was necessary to add `trio_typing`. Do you see an issue with that?

Reply:
no